docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,237 @@
+   - List the Windows IP addresses or DNS names under `non_ad_hosts`.
 
-In order to gain access to a remote desktop, a Teleport user needs to have the
+   Teleport can't automatically discover Windows desktops without Active Directory. 
+   Therefore, you must specify the Windows IP addresses or DNS names that you want to 
+   enroll using the Teleport configuration file or use the Teleport API  
+   to build your own integration. You can find an example of API integration on 
+   [GitHub](https://github.com/gravitational/teleport/tree/master/examples/desktop-registration).
+
+1. (Optional) Add labels for the Windows Desktop Service to the configuration file.
+
+   You can attach labels to Windows computers by matching to their hostnames.
+   For example, the following adds the `cloud: ec2` label to computers that have 
+   EC2 private IP DNS names matching `.us-east-2.compute.internal`:
+
+   ```diff
+   version: v3
+   teleport:
+     nodename: windows.teleport.example.com
+     proxy_server: teleport-proxy.example.com:443
+   windows_desktop_service:
+     enabled: yes
+     non_ad_hosts:
+   -    - 192.0.2.156
+   +    - ip-192-0-2-156.us-east-2.compute.internal
+   + host_labels:
+   +    - match: '.*\.us-east-2.compute.internal'
+   +      labels:
+   +        cloud: ec2
+   ```
+
+1. (!docs/pages/includes/start-teleport.mdx service="the Teleport Windows Desktop Service"!)
+
+## Step 3/4. Configure remote Windows desktop access
+
+To access a remote desktop, a Teleport user must have a role with the
 appropriate permissions for that desktop.
 
-Create the file `windows-desktop-admins.yaml`:
-
-```yaml
-kind: role
-version: v6
-metadata:
-  name: windows-desktop-admins
-spec:
-  allow:
-    windows_desktop_labels:
-      "*": "*"
-    windows_desktop_logins: ["Administrator", "alice"]
-```
+To configure a role for desktop access:
 
-You can restrict access to specific hosts by defining values for
-`windows_desktop_labels`, and adjust the array of usernames this role has access
-to in `windows_desktop_logins`.
+1. Sign in to the Teleport cluster from your administrative workstation to authenticate your identity.
 
-<Admonition type="warning" title="RBAC Configuration">
-Ensure that each Teleport user is only assigned Windows logins that they should
-be allowed to access.
-</Admonition>
+1. Create a `windows-desktop-admins.yaml` file to define a new role:
 
-Apply the new role to your cluster:
+   ```yaml
+   kind: role
+   version: v6
+   metadata:
+     name: windows-desktop-admins
+   spec:
+     allow:
+       windows_desktop_labels:
+         "*": "*"
+       windows_desktop_logins: ["Administrator", "alice"]
+   ```
 
-```code
-$ tctl create -f windows-desktop-admins.yaml
-```
+   In this file:
+
+   - Set `windows_desktop_labels` to specify any labels you want to use to allow or deny access to specific hosts.
+   - Set `windows_desktop_logins` to specify the Windows logins that Teleport users assigned to the role
```

Comment:
Maybe we should also expand on automatic user creation (the logins you specify in `windows_desktop_logins` don't have to exist ahead of time on the Windows machine, Teleport can create them on demand).

This is only true for non-AD hosts and only if RBAC allows it, and it's not well documented.